apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp

```diff
@@ -105,26 +105,39 @@ static void utc_media_MediaRecorder_setDataSource_n(void)
 
 static void utc_media_MediaRecorder_setVolume_p(void)
 {
+	uint8_t prev, volume;
 	MediaRecorder mr;
 	unique_ptr<FileOutputDataSource> dataSource = unique_ptr<FileOutputDataSource>(new FileOutputDataSource(channels, sampleRate, pcmFormat, filePath));
 	mr.create();
 	mr.setDataSource(std::move(dataSource));
 	mr.prepare();
-	TC_ASSERT_EQ("utc_media_mediarecorder_setVolume", mr.setVolume(10), RECORDER_OK);
+	mr.getVolume(&prev);
+
+	TC_ASSERT_EQ_CLEANUP("utc_media_mediarecorder_setVolume", mr.setVolume(10), RECORDER_OK, goto cleanup);
+	mr.getVolume(&volume);
+	TC_ASSERT_EQ_CLEANUP("utc_media_mediarecorder_setVolume", volume, 10, goto cleanup);
+
+cleanup:
+	mr.setVolume(prev);
 	mr.unprepare();
 	mr.destroy();
 	TC_SUCCESS_RESULT();
 }
 
 static void utc_media_MediaRecorder_setVolume_n(void)
 {
+	uint8_t prev;
 	MediaRecorder mr;
 	unique_ptr<FileOutputDataSource> dataSource = unique_ptr<FileOutputDataSource>(new FileOutputDataSource(channels, sampleRate, pcmFormat, filePath));
 	TC_ASSERT_EQ("utc_media_mediarecorder_setVolume", mr.setVolume(10), RECORDER_ERROR_NOT_ALIVE);
 	mr.create();
 	mr.setDataSource(std::move(dataSource));
 	mr.prepare();
-	TC_ASSERT_EQ("utc_media_mediarecorder_setVolume", mr.setVolume(11), RECORDER_OK);
+	mr.getVolume(&prev);
+	TC_ASSERT_EQ_CLEANUP("utc_media_mediarecorder_setVolume", mr.setVolume(11), RECORDER_OK, goto cleanup);
+
+cleanup:
+	mr.setVolume(prev);
```

Comment:
@junmin-kim Does it need to do set prev in situations where it does not set?

Reply:
This test try to set volume as 11, but set 10(max value).
So it is needed to restore volume